tools/wt_binary_decode.py

```diff
@@ -555,6 +554,61 @@ def row_decode(p, b, pagehead, blockhead, pagestats):
         finally:
             p.end_cell()
 
+def extlist_decode(p, b, pagehead, blockhead, pagestats):
+    WT_BLOCK_EXTLIST_MAGIC = 71002       # from block.h
+    # Written by block_ext.c
+    okay = True
+    cellnum = -1
+    lastoff = 0
+    p.rint_v('extent list follows:')
+    while True:
+        cellnum += 1
+        cellpos = b.tell()
+        if cellpos >= pagehead.mem_size:
+            p.rint_v(f'** OVERFLOW memsize ** memsize={pagehead.mem_size}, position={cellpos}')
+            #return
+        p.begin_cell(cellnum)
+
+        try:
+            off = b.read_packed_uint64()
+            size = b.read_packed_uint64()
+            extra_stuff = ''
+            if cellnum == 0:
+                extra_stuff += '  # magic number'
+                if off != WT_BLOCK_EXTLIST_MAGIC or size != 0:
+                    extra_stuff = f'  # ERROR: magic number did not match expected value=' + \
+                        '{WT_BLOCK_EXTLIST_MAGIC}'
+                    okay = False
+            else:
+                if off < lastoff:
+                    extra_stuff = f'  # ERROR: list out of order'
+                    okay = False
+
+                # We expect sizes and positions to be multiples of
+                # this number, it is conservative.
+                multiple = 256
+                if off % multiple != 0:
+                    extra_stuff = f'  # ERROR: offset is not a multiple of {multiple}'
+                    okay = False
+                if size % multiple != 0:
+                    extra_stuff = f'  # ERROR: offset is not a multiple of {multiple}'
+                    okay = False
+            if off == 0:
+                # Corresponds to code in block_ext.c
+                extra_stuff += '  # end of list'
+                if size == 0:
+                    extra_stuff += ', not a final checkpoint'
+                elif size == 1:
+                    extra_stuff += ', final checkpoint'
```

Comment:
In this case, we will also get the "ERROR: size is not a multiple of 256" message, from above. But here it's not an error.